src/components/top-bar.js

```diff
@@ -32,10 +41,52 @@ const useStyles = makeStyles(theme => ({
     }
 }));
 
+const StyledMenu = withStyles({
+    paper: {
+        border: '1px solid #d3d4d5',
+    },})(props => (
+        <Menu
+            elevation={0}
+            getContentAnchorEl={null}
+            anchorOrigin={{
+                vertical: 'bottom',
+                horizontal: 'center',
+            }}
+            transformOrigin={{
+                vertical: 'top',
+                horizontal: 'center',
+            }}
+            {...props}
+        />
+));
+
+const StyledMenuItem = withStyles(theme => ({
+    root: {
+        '&:focus': {
+            backgroundColor: theme.palette.primary.main,
+            '& .MuiListItemIcon-root, & .MuiListItemText-primary': {
```

Comment:
not sure if we are supposed to use the Mui* classes ? Shouldn't we use our own classes ?

Reply:
This is documented in the following example: https://material-ui.com/components/menus/#customized-menus "

Reply:
So it's ok